configure.ac

```
@@ -472,6 +476,10 @@ if test "x$CXXFLAGS_overridden" = "xno"; then
 
   if test x$suppress_external_warnings != xno ; then
     AX_CHECK_COMPILE_FLAG([-Wdocumentation],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wdocumentation"],,[[$CXXFLAG_WERROR]])
+    AX_CHECK_COMPILE_FLAG([-Wzero-as-null-pointer-constant],[
+      WARN_CXXFLAGS="$WARN_CXXFLAGS -Wzero-as-null-pointer-constant"
+      AC_DEFINE(HAVE_ZERO_AS_NULL_POINTER_CONSTANT, 1, [Define this symbol if -Wzero-as-null-pointer-constant is available])
+    ],,[[$CXXFLAG_WERROR]])
```

One pair of `[]` is redundant:
```suggestion
    AX_CHECK_COMPILE_FLAG([-Wzero-as-null-pointer-constant], [
      WARN_CXXFLAGS="$WARN_CXXFLAGS -Wzero-as-null-pointer-constant"
      AC_DEFINE(HAVE_ZERO_AS_NULL_POINTER_CONSTANT, 1, [Define this symbol if -Wzero-as-null-pointer-constant is available])
    ], [], [$CXXFLAG_WERROR])
```